src/PersonLink.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import classNames from 'classnames';
+import { useParams, Link } from 'react-router-dom';
+import { Person } from './types';
+
+const findParent = (
+  person: Person,
+  parentType: string,
+  name: string,
+  peopleList: Person[],
+) => {
+  const parent = peopleList
+    .find(personToFind => personToFind.name === name);
+
+  if (parent) {
+    return (
+      <Link
+        className={classNames(
+          { 'has-text-danger': parentType === 'mother' },
+        )}
+        to={`/people/${parent.slug}`}
+      >
+        {parentType === 'mother'
```

Comment:
It is better to put such constructions into separate variables and substitute the already calculated result into the component.